components/brave_rewards/browser/brave_rewards_data_remover_delegate.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_REWARDS_DATA_BRAVE_REWARDS_DATA_REMOVER_DELEGATE_H_
+#define BRAVE_BROWSER_REWARDS_DATA_BRAVE_REWARDS_DATA_REMOVER_DELEGATE_H_
+
+#include <memory>
+
+#include "base/callback_forward.h"
+#include "base/memory/ref_counted.h"
+#include "base/memory/weak_ptr.h"
+#include "base/optional.h"
+#include "base/synchronization/waitable_event_watcher.h"
+#include "base/task/cancelable_task_tracker.h"
+#include "chrome/browser/profiles/profile.h"
+#include "components/keyed_service/core/keyed_service.h"
+#include "content/public/browser/browsing_data_remover.h"
+#include "chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h"
+
+class Profile;
+
+namespace content {
+class BrowserContext;
+}
+
+class BraveRewardsDataRemoverDelegate
```

Comment:
this should not be `Rewards`. We might also use this for other types of data in the future so it should be a generic `BraveBrowsingDataRemoverDelegate`

Reply:
Renamed